src/js/components/devices/troubleshootdialog.js

```diff
@@ -40,8 +40,8 @@ const ConnectionIndicator = isConnected => {
 };
 
 const tabs = {
-  terminal: { link: 'session logs', title: ConnectionIndicator, value: 'terminal' },
-  transfer: { link: 'file transfer logs', title: () => 'File transfer', value: 'transfer' }
+  terminal: { link: 'session logs', title: ConnectionIndicator, value: 'terminal', needsWriteAccess: true, needsTroubleshoot: true },
+  transfer: { link: 'file transfer logs', title: () => 'File transfer', value: 'transfer', needsWriteAccess: false, needsTroubleshoot: false }
```

Comment:
Why file transfer doesn't need troubleshoot?

Reply:
Yeah, that's not ideal... Troubleshoot is needed on the device and in the account, but downloading a file should also work for read only users... I think the access restriction might be a bit too fine granular in this case, but on the other hand I think it is a valid point...